modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -204,7 +204,7 @@ More configuration options can be found in https://www.keycloak.org/docs/latest/
 ====
 
 If your Identity Provider is correctly configured (see the section _Configure the Identity Provider_), you are done.
```

Comment:
If your Identity Provider is correctly configured (see the section _Configure the Identity Provider_), you are done. => Once your Identity Provider is correctly configured (see the section _Configure the Identity Provider_), there are no more activities on this part.

Reply:
I proposed a mix between the 2 phrases. "you are done" didnt seem appropriate.

Reply:
I added this.